### PR TITLE
Update jsonDataProvider example to include the POJO.

### PR DESCRIPTION
**Pull request by ILikeToNguyen**
Update jsonDataProvider example to include the POJO.

### CODE REVIEW

File: sample-files/jsonDataProviderUserClass.java [hunk 1]
```diff
@@ -0,0 +1,90 @@
+public class USER {
```
Comment (ILikeToNguyen) [inline sample-files/jsonDataProviderUserClass.java:1]: USER is the example class name we already have in place.
This is a class purely in here for website documentation so we don't waste extra space with package, imports, etc.

File: sample-files/jsonDataProviderUserClass.java [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+    public void setSize(int size) {
+        this.size = size;
+    }
+
+    public AREA_CODE[] getAreaCode() {
+        return (areaCode == null) ? null : Arrays.copyOf(areaCode, areaCode.length);
+    }
+
+    public void setAreaCode(AREA_CODE[] areaCode) {
```
Comment (ILikeToNguyen) [inline sample-files/jsonDataProviderUserClass.java:70]: Disregard this

File: sample-files/jsonDataProviderUserClass.java [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+            BANK bank, String phoneNumber, int preintTest) {
+        this.setName(name);
+        this.setPassword(password);
+        this.setAccountNumber(accountNumber);
+        this.setAmount(amount);
+        this.setSize(size);
+        this.setAreaCode(areaCode);
+        this.setBank(bank);
+        this.setPhoneNumber(phoneNumber);
+        this.setPreintTest(preintTest);
```
Comment (sebady) [inline sample-files/jsonDataProviderUserClass.java]: where is this setter in the USER example?

**Comment by sebady** [comment]
merged

**Pull request closed without merging**